### PR TITLE
zh trans host-aliases.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: /docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/host-aliases.md]: 改为中文文档的链接

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+short_description: >
+  A HostAliases is a mapping between the IP address and hostname to be injected into a Pod's hosts file.
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/host-aliases.md]: 这句话翻译一下

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+HostAliases 是要注入 Pod 的 hosts 文件的 IP 地址和主机名之间的映射。
+
+<!--more-->
+
+<!--
+[HostAliases](/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
+-->
+
+[HostAliases](/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) 是一个可选的主机名和 IP 地址列表，如果指定，它们将被注入 Pod 的 hosts 文件中。 这仅适用于非 hostNetwork 的 Pod。
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/host-aliases.md]: 改为中文文档的链接 :)

**Comment by markthink** [comment]
@congfairy 冲突需要解决一下

**Pull request merged**